asad/interactive.py

```diff
@@ -341,7 +341,7 @@ def do_set_wavelength_start(self, arg):
         except IndexError as index_error:
             error_print('invalid wavelength start, wavelength start not set')
             error_print(str(index_error))
-            pass
+            raise index_error
```

Comment:
This was an earlier requirement, use defaults read in from the file if there is an error, check line 376.
We should review and decide if that's the required behavior now.